tests/conftest.py

```diff
@@ -23,7 +23,7 @@
 
 # Those variables probably should go to main NeMo configuration file (config.yaml).
 __TEST_DATA_FILENAME = "test_data.tar.gz"
-__TEST_DATA_URL = "https://github.com/NVIDIA/NeMo/releases/download/v0.11.0/"
+__TEST_DATA_URL = "https://github.com/NVIDIA/NeMo/releases/download/v1.0.0rc1/"
```

Comment:
You should double check with oleksii before making this change

Reply:
Checked w/ Oleksii during the meeting, should be fine.